bsst/ville_config.py

```diff
@@ -14,5 +14,5 @@
 SAVE_FILE_PATH = os.path.join(os.path.abspath(__file__).split('py')[0], 'act_villes')
 FILE_OPEN_FN = bsst_io.open_fn
 
-SUCCESS_DATES = slice(np.datetime64('2020-12-01'), np.datetime64('2021-02-28'))
+SUCCESS_DATES = slice(np.datetime64('2021-07-01'), np.datetime64('2021-10-01'))
```

Comment:
@kleined Note that I changed these dates so that the site activation demo notebook works with the new demo data.